web_interface/astpp/application/controllers/currencyupdate.php

```diff
@@ -24,72 +24,108 @@
 
 
 class Currencyupdate extends CI_Controller {
+
+    public $fp = "";
+
 	function __construct() {
 		parent::__construct ();
 		// if(!defined( 'CRON' ) )
 		// exit();
 		$this->load->model ( "db_model" );
 		$this->load->library ( "astpp/common" );
+        $this->fp = fopen("/var/log/astpp/astpp-currency.log", "a+");
```

Comment:
Thanks for the update @AlexHeylin 

Would you please use variable for log path in place of using static path? 

Use Common_model::$global_config ['system_config'] ['log_path']

Thanks